scanline/ScannerController.swift

```
@@ -50,7 +50,7 @@ class ScannerController: NSObject, ICScannerDeviceDelegate {
         guard error == nil else {
             logger.log("Error received while attempting to open a session with the scanner.")
             delegate?.scannerControllerDidFail(self)
-            return
+            exit(1)
```

In a batch scan, would this change the current behavior? Wondering if there's an issue where you're scanning several items in a batch and then one of them fails -- would it exit now instead of bailing on that particular item and continuing on?

---

Good point! I haven't thought of that use case. I just found it confusing that the command did not exit when encountering an error. Instead it kept running forever. To be safe I added a fallback to return for the batch scan case now.